ScarletEngine/Sources/Gameplay/Private/World.cpp

```diff
@@ -15,7 +15,7 @@ namespace ScarletEngine
 
 		SceneProxy* ProxyPtr = &RenderSceneProxy;
 		AddSystem<const Transform, const StaticMeshComponent>("Draw Static Meshes")
-			.Each([ProxyPtr](const EID, const Transform& Trans, const StaticMeshComponent& SMC)
+			.Each([ProxyPtr](double, const EID, const Transform& Trans, const StaticMeshComponent& SMC)
```

Comment:
Is that really needed?

Reply:
Yes because systems are now expecting functions with the signature `(double, EID, ...)` so that delta times can be passed directly without having to get it from the world each frame.